### ISSUE TITLE
Adding semicolon at the end of schema block

### ISSUE THREAD

**Issue by marcinkrzeminski** [panoply/vscode-liquid#13]
Hi,

This extension is pretty great. However, I'm having an issue when formatting the schema block. It adds a semicolon after the last closing curly bracket and when I try to sync (using Slater) the template it says `Invalid JSON in tag 'schema'`. It happens with all but this extension being disabled.

Also, it adds extra space before and after the block. 

Here's an example with before and after

```
{% schema %}
{
  "name": "Customizer",
  "blocks": [
    {
      "name": "Intro",
      "type": "intro",
      "settings": [
        {
          "type": "richtext",
          "id": "copy",
          "label": "Copy"
        }
      ]
    }
  ]
}
{% endschema %}
```

```
{% schema %}

{
  "name": "Customizer",
  "blocks": [
    {
      "name": "Intro",
      "type": "intro",
      "settings": [
        {
          "type": "richtext",
          "id": "copy",
          "label": "Copy"
        }
      ]
    }
  ]
};

{% endschema %}
```

**Comment by panoply** [comment]
Hey @marcinkrzeminski

I never heard of slater, I roll my own build environments for Shopify themes. The reason a semicolon is added is because VS Code is treating the content within the `schema` block as `JSON` – There should be a setting within the VS Settings that allows you to prevent the semicolon being added. I will look at applying a fix in later versions and PR's are welcome too.  

Regarding the extra space added before and after the `schema` block, this is an intentional design implementation. I may look at making this optional in a future versions.

**Comment by marcinkrzeminski** [comment]
Hey @panoply 

What's weird the semicolon is added when I hit save second time after creating a schema block. Once that done it adds the semicolon all the time.

**Comment by panoply** [comment]
@marcinkrzeminski You can now choose if you wish to add a `brace_block` rule to prevent double newline spacing between section blocks. By default the extension will ignore newline spacing on `{% schema %}` content (meaning no newlines) but with `{% stylesheet %}` or `{% javascript %} the newlines will be preserved and act as usual. Refer to the new rule `brace_block` in the readme.

In regards to your extra semicolon issue, try adding this to your VS Code settings and let me know if it works:

```json
"html.validate.scripts": false
```

**Comment by panoply** [comment]
@marcinkrzeminski This is fixed in version 1.7.3. Added the `no_semicolon` ruleset.
